feathery-build.patch

```diff
@@ -0,0 +1,58 @@
+From 5a3520540276a6fd06f7c88e571b6462978e3eab Mon Sep 17 00:00:00 2001
+From: Jammyjamjamman <jdsherratt3@gmail.com>
+Date: Sat, 20 Jun 2020 19:30:01 +0100
+Subject: [PATCH] Add ftp fixes
```

Comment:
Where did this patch come from?

Reply:
This: https://github.com/MegaGlest/megaglest-source/commit/5a3520540276a6fd06f7c88e571b6462978e3eab